### PR TITLE
added reversing no using py3

### PR DESCRIPTION
**Pull request by agnipanda**

### CODE REVIEW

File: linked_list_in_cpp/main.cpp [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#include "iostream.h"
```
Comment (t2013anurag) [inline linked_list_in_cpp/main.cpp]: move file to existing linked_list directory.

File: linked_list_in_cpp/main.cpp [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+using namespace std;
```
Comment (t2013anurag) [inline linked_list_in_cpp/main.cpp]: rename file to linked_list_in_cpp.cpp

File: reverse_no_py3/revno.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+n=input("Enter a number:")
```
Comment (t2013anurag) [inline reverse_no_py3/revno.py]: move file to existing reverse a no directory.

File: reverse_no_py3/revno.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+l=list(n)
+l.reverse()
+n="".join(l)
+print(n)
```
Comment (t2013anurag) [inline reverse_no_py3/revno.py]: add a new line

File: simple_encryption_algo_python3/decrypt.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+f1=open("encrypted_file.txt","r")
```
Comment (t2013anurag) [inline simple_encryption_algo_python3/decrypt.py]: move to encryption_algorithms directory, rename file or create a sub directory inside, with proper documentation.

File: simple_encryption_algo_python3/decrypt.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+s=f1.read()
+f1.close()
+s=list(s)
+for i in range(len(s)):
+	s[i]=chr(ord(s[i])-len(s)+i)
+s.reverse()
+s="".join(s)
+f2=open("decrypted_file.txt","w+")
+f2.write(s)
+f2.close()
```
Comment (t2013anurag) [inline simple_encryption_algo_python3/decrypt.py]: new line.

File: simple_encryption_algo_python3/encrypt.py [hunk 7]
```diff
@@ -0,0 +1,11 @@
+f1=open("file_before_enc.txt","r")
```
Comment (t2013anurag) [inline simple_encryption_algo_python3/encrypt.py]: same as above,

File: simple_encryption_algo_python3/file_before_enc.txt [hunk 8]
```diff
@@ -0,0 +1 @@
+hello world...!!
```
Comment (t2013anurag) [inline simple_encryption_algo_python3/file_before_enc.txt]: move inside existing directories.

**Comment by t2013anurag** [comment]
Also create a github issue and reference them for each of your files.

File: linked_list_in_cpp/main.cpp (hunk 1 above)
Comment (agnipanda) [reply linked_list_in_cpp/main.cpp]: done

File: linked_list_in_cpp/main.cpp (hunk 2 above)
Comment (agnipanda) [reply linked_list_in_cpp/main.cpp]: done

File: reverse_no_py3/revno.py (hunk 3 above)
Comment (agnipanda) [reply reverse_no_py3/revno.py]: done

**Pull request merged**